tests/plugins/mep/test_reader.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pytest
+
+from pyaerocom.plugins.mep.reader import ReadMEP
+
+MEP_PATH = Path("/lustre/storeA/project/aerocom/aerocom1/AEROCOM_OBSDATA/MEP/download")
+
+STATION_NAMES = ("1478A", "2706A", "3377A")
+
+VARS_DEFAULT = {"concco", "concno2", "conco3", "concpm10", "concpm25", "concso2"}
+VARS_PROVIDED = VARS_DEFAULT | {"vmro3", "vmro3max", "vmrno2"}
+
+
+needs_mep_path = pytest.mark.xfail(not MEP_PATH.is_dir(), reason=f"needs access to {MEP_PATH}")
+
+
+@pytest.fixture(scope="module")
+def reader() -> ReadMEP:
+    return ReadMEP(data_dir=str(MEP_PATH))
+
+
+@pytest.fixture()
+def station_files(station: str) -> list[Path]:
+    files = sorted(MEP_PATH.rglob(f"mep-rd-{station}-*.nc"))
+    assert files, f"no files for {station}"
+    return files
+
+
+def test_reader_error():
+    with pytest.raises(ValueError) as e:
+        ReadMEP(data_dir=None)
+
+
+@needs_mep_path
+def test_DATASET_NAME(reader: ReadMEP):
+    assert reader.DATASET_NAME == "MEP"
+
+
+@needs_mep_path
+def test_DEFAULT_VARS(reader: ReadMEP):
+    assert set(reader.DEFAULT_VARS) >= VARS_DEFAULT
+
+
+@needs_mep_path
+def test_files(reader: ReadMEP):
+    assert reader.files, "no stations files found"
+    assert len(reader.files) >= 167482, "found less files than expected"
```

Comment:
I'm admittedly a little worried that these tests are rather brittle. If anything changes in the data, CI will crash. These data aren't really stable enough nor have we had them long enough to know that this won't happen. We can keep it for now if you think it's worth it, but in the future maybe we want to come up with a more robust test. For example, we could check that reader.files contains only elements of type str. Right now we're kind of "testing the implementation", not the functionality